client/html/layouts/partials/_head.html

```diff
@@ -25,6 +25,7 @@
     unlock_next: "ON_CORRECT",
     locale: "en",
     audio_recorder_timeout: 100,
+    authoring_tool_preview_settings: ['unlock_next=ON_CORRECT'],
```

Comment:
Plus it wouldn't really be a better 'default', it would just be the hard coded value that better aligns with how we think it should work this week.